v2.0/show-create-sequence.md

```diff
@@ -0,0 +1,71 @@
+---
+title: SHOW CREATE SEQUENCE
+summary: The SHOW CREATE SEQUENCE statement shows the CREATE SEQUENCE statement that would create a copy of the specified sequence.
+toc: false
+---
+
+The `SHOW CREATE SEQUENCE` [statement](sql-statements.html) shows the `CREATE SEQUENCE` statement that would create a copy of the specified table.
+
+<div id="toc"></div>
+
+## Required Privileges
+
+The user must have any [privilege](privileges.html) on the target sequence.
+
+## Synopsis
+
+{% include sql/{{ page.version.version }}/diagrams/show_create_sequence.html %}
+
+## Parameters
+
+Parameter | Description
+----------|------------
+`sequence_name` | The name of the sequence for which to show the `CREATE SEQUENCE` statement.
+
+## Response
+
+Field | Description
+------|------------
+`Sequence` | The name of the table.
+`CreateSequence` | The [`CREATE SEQUENCE`](create-sequence.html) statement for creating a copy of the specified sequence.
+
+## Example
+
+~~~ sql
```

Comment:
I think it's sufficient to show one `create sequence` and `show create sequence`. I'd get rid of `create sequence customer_seq` and `show create sequence customer_seq`.

Reply:
Done.